src/neptune/handler.py

```diff
@@ -612,11 +617,16 @@ def download(self, destination: str = None) -> None:
                 composed from field name and extension (if present).
                 If `destination` is a path to a file, the file will be downloaded under the specified name.
                 Defaults to `None`.
+            progress_bar: (bool or Type of progress bar, optional): progress bar to be used while downloading assets.
+                If `None` or `True` the default tqdm-based progress bar will be used.
+                If `False` no progress bar will be used.
+                If a type of progress bar is passed, it will be used instead of the default one.
```

Comment:
How would you recommend to instruct the end-user on how to accomplish this? Is it correct to say that they can define a class that inherits from `ProgressBarCallback` and pass its type as the argument? (Asking also for docs purposes)

Reply:
Yes, just like in the example with `click` in the docstrings. You can pass this `ClickProgressBar`.
Maybe link it somehow?

Reply:
Very good; your example did make it super clear. Just wanted to double-check that I've got it right and that this alias didn't change anything on the user-end.